docker/Dockerfile

```
@@ -1,14 +1,12 @@
-FROM ubuntu:18.04
+FROM alpine:3.10
 
 ENV APP_VERSION=5.9.0
 ENV PATH="/mattermost-push-proxy/bin:${PATH}"
 
 # Install some needed packages
-RUN apt-get update \
-    && apt-get -y --no-install-recommends install \
-      curl \
-      wget \
-      netcat \
+RUN apk update \
```

```suggestion
RUN apk add --no-cache\
```
(update is obsolete with `--no-cache`)

---

Good point!

---

Good point!